src/i18n/ua-ua/index.ts

```diff
@@ -1,11 +1,11 @@
 export default {
   language: 'Мова',
   ddos: {
-    description: 'Атака DDoS робить величезну кількість запитів на ворожі сайти, чим спричиняє їхню величезну завантаженість і виводить їх з ладу. В наслідку атаки DDoS, сервери не можуть працювати в штатному режимі / лагають, або взагалі не працюють.',
-    coordinators: 'Для координацій цілей: https://t.me/incourse911 ',
+    description: 'Атака DDoS робить величезну кількість запитів на ворожі сайти, чим спричиняє їхню величезну завантаженість і виводить їх з ладу. В наслідку атаки DDoS, сервери не можуть працювати в штатному режимі / лагають, або взагалі не працюють',
+    coordinators: 'Для координацій цілей: https://t.me/incourse911',
```

Comment:
Andrii, I saw your readme with resources to contribute to the cause, It would be wonderful to also have it in english :pray: 
If you are busy and don't have time to translate I can help you.
warning: I don't speak ukranian I could translate from google translate and send you a PR so you can check if the info is correct

Reply:
@belueg no worries, I know English very well. I would consider ONLY English, but... Anyway I will update my README into English only so far.